modules/virt-virtctl-commands.adoc

```diff
@@ -52,8 +52,14 @@ the specified port of the node.
 |`virtctl console <vmi_name>`
 |Connect to a serial console of a virtual machine instance.
 
-|`virtctl vnc <vmi_name>`
-|Open a VNC connection to a virtual machine instance.
+|`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
+|Open a VNC connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a Virtual Network Client (VNC) which requires a remote viewer on your local machine.
```

Comment:
I would suggest spelling out the acronym VPN when it first occurs in the sentence rather than after its first occurrence. 
"Open a Virtual Network Client (VNC) connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a VNC connection which requires a remote viewer on your local machine."

Reply:
Made the change. Please review.